README.md

```diff
@@ -10,6 +10,7 @@ Primer Spec is built on top of the wonderful [Primer theme](https://github.com/p
 - [Usage](#usage)
   - [Using with Jekyll](#using-with-jekyll)
```

Comment:
Something that's confusing about the README are the terms "Jekyll" and "Markdown".  I think you're using them interchangeably.  Perhaps "Using with Markdown and Jekyll"?  This confusion also caught me when reading the opening description of the README.

Reply:
Hmm, I see how this could be confusing. This project is technically a Jekyll theme (since it can be rendered with the Jekyll Remote Theme plugin), but will also be distributed as a standalone JavaScript plugin that formats any compatible HTML page.

Adding to the confusion: Jekyll has a built-in Markdown-to-HTML converter, but it's not necessary to use it. All of our project specs are written in Markdown for easy maintenance.

Maybe it would be cleaner to simply describe Primer Spec as a Jekyll theme with support for Markdown? (Especially since we probably want to recommend the Jekyll+Markdown approach to new courses.) Then, deeper in the README, we could describe how it could also be used without Jekyll.

Reply:
I think you're right — we should probably always recommend new users to start with Jekyll (and Markdown). The HTML/JS plugin covers a slightly more niche use-case.